scripts/utils.py

```diff
@@ -21,11 +21,6 @@
 import subprocess
 from typing import Sequence
 
-BAZEL_FILTERS = [
-    r'Loading: [0-9]+ packages loaded',
-    r'.*Using python binary from PYTHON_BIN = .*'
```

Comment:
Any way to prevent the logs from being flooded by this message?

Reply:
We can add https://docs.bazel.build/versions/master/command-line-reference.html#flag--ui_event_filters

Reply:
Sounds good to me!

Reply:
I'll take a look at this later. Currently dealing with integration hell :fire: :imp: :fire:

Reply:
> The set of supported event kinds include INFO, DEBUG, ERROR and more.

Suuuuuuper helpful Bazel :roll_eyes: But anyway I found the option. Unfortunately it doesn't let me silence the loading messages, so we still have to use the separate flag for those.